### DISCUSSION TITLE
Where is unsigned.io?

### DISCUSSION THREAD

**Discussion by plongk** [markqvist/Reticulum#523] [General]
The unsigned.io site is resolving, but not responding for me ... neither through numerous VPN endpoints nor via centralops.net.  I can also see that there is almost no repository activity by Mark since 07-Jun-2024, and nothing (that I could find) since 29-Jun.

**Comment by Sudo-Ivan** [comment]
He is currently away. Here is a mirror of the site for the time being: https://reticulum.betweentheborders.com/

**Comment by liamcottle** [comment]
Another option is the wayback machine, which includes the most recent post:
https://web.archive.org/web/20240722075626/https://unsigned.io/

**Comment by markqvist** [comment]
I had a severe power failure at the site where I run my webserver, and was not able to get it fixed for some time. Everything is now sorted again :)
